src/topology/path_connected.lean

```diff
@@ -94,6 +94,8 @@ protected lemma continuous : continuous γ :=
 @[simp] protected lemma target : γ 1 = y :=
 γ.target'
 
+@[simp] lemma to_continuous_map_apply {t : I} : γ.to_continuous_map t = γ t := rfl
```

Comment:
I'd be tempted to state this about coercions
```suggestion
@[simp] lemma coe_to_continuous_map : ⇑γ.to_continuous_map = γ := rfl
```

Reply:
Actually, might as well do this for `continuous_map.homotopy` as well.